tests/scripts/k8s-vagrant-multi-node.sh

```diff
@@ -43,7 +43,7 @@ function copy_images() {
     if [[ "$1" == "" || "$1" == "ceph" ]]; then
       echo "copying ceph images"
       copy_image_to_cluster "${BUILD_REGISTRY}/ceph-amd64" rook/ceph:master
-      copy_image_to_cluster ceph/ceph:v15 ceph/ceph:v15
+      copy_image_to_cluster ceph/ceph:v16 ceph/ceph:v16
```

Comment:
Unrelated to this PR, I wonder if anyone is still using this script?

Reply:
I feel like this also isn't necessary. I don't use this script, but I do use k8s-vagrant-multi-node, and it can pull images from the internet just fine...